deepspeed/module_inject/auto_tp.py

```diff
@@ -17,9 +17,11 @@
 from deepspeed.module_inject.tp_shard import get_shard_size, get_shard_size_list
 
 
-def move(tensor, device):
+def move(tensor, device, keep_module_on_host=False):
```

Comment:
Won't it be simpler to modify callers to pass device='cpu' when `keep_module_on_host=True`?

Reply:
Done